platform/core/src/index.test.js

```diff
@@ -10,7 +10,8 @@ describe('Top level exports', () => {
       'HotkeysManager',
       'ServicesManager',
       //
-      'createUiNotificationService',
+      'createUINotificationService',
+      'createUIModalService',
```

Comment:
Note: this doesn't match the casing of the same method within `UIModalService/index.js`

Reply:
Done! thank you.